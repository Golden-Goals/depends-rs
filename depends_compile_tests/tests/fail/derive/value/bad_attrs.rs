use depends::derives::Value;

#[derive(Value)]
#[depends(unhashable = true)]
struct Foo {
    bar: Vec<usize>,
}

fn main() {}
