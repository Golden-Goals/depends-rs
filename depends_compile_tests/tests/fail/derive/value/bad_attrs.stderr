error: expected `,`
 --> tests/fail/derive/value/bad_attrs.rs:4:22
  |
4 | #[depends(unhashable = true)]
  |                      ^
